requirements/base.in

```diff
@@ -9,6 +9,7 @@ channels_redis==2.4.0
 CherryPy<18.2
 colour==0.1.5
 cookies<2.3
+cryptography>=3.2
```

Comment:
Although i know it would be a good idea in general to pin dependencies like this, but since OCIM is our deployment tool, could we pin to the exact dependency, since it would install 4.0 as well if requirements allows it.

Reply:
Sure, I'll make that change and reupload the PR.